cmakelib/bindings/bindings.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package bindings implements CMake-style variable bindings.
```

Comment:
Maybe include a pointer to CMake docs on this.  (Nice to have the link below already for Get, since that's where it's more confusing, but a top-level link to a relevant cmake.org page here would be good too I think)